code/site/components/com_pages/dispatcher/router/resolver/site.php

```diff
@@ -66,7 +68,7 @@ public function resolve(ComPagesDispatcherRouterInterface $router)
             //Configure the template
             if(isset($config['template']) || isset($config['template_config']))
             {
-                if(isset($config['template']) && $config['template'] != false)
+                if(isset($config['template']))
```

Comment:
@johanjanssens This `if` statement is repeated on the next line so can be removed here.

Reply:
@stevenrombauts Good catch. Done.